### PR TITLE
Zonemaster - EN/FR duplication

### PR DESCRIPTION
**Pull request by OVH-Team-Guides**

### CODE REVIEW

File: pages/web/domains/dns_zonemaster/guide.es-es.md [hunk 1]
```diff
@@ -38,42 +36,46 @@ updated: 2022-09-12
...
+>
+> ![Captura de pantalla de la página de resultados de Zonemaster para el dominio "mydomain.ovh". Se desarrolla la sección "Direcciones".](images/zonemaster03.png){.thumbnail}
 
 ### Resultado
 
-Una vez validado el formulario, los resultados se clasifican por código de color:
+Una vez validado el formulario, los resultados se muestran por grupo de pruebas. Las pruebas se clasifican por su severidad.
+
+- **Error**: esta parte presenta errores o elementos faltantes que pueden causar un fallo de funcionamiento.
+- **Advertencia**: Esta parte es funcional, pero merece una atención especial. La herramienta ha detectado que este parámetro tiene características que no encajan en el estándar de su categoría, sin bloquear su funcionamiento.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.es-es.md]:
```suggestion
- **Advertencia**: esta parte es funcional, pero merece una atención especial. La herramienta ha detectado que este parámetro tiene características que no encajan en el estándar de su categoría, sin bloquear su funcionamiento.
```

File: pages/web/domains/dns_zonemaster/guide.es-es.md [hunk 2]
```diff
@@ -38,42 +36,46 @@ updated: 2022-09-12
...
-- **Rojo**: Esta parte presenta errores o elementos que faltan y que pueden provocar un fallo de funcionamiento. 
-- **Azul**: se trata de una simple información, sin consecuencias particulares sobre el funcionamiento del nombre de dominio.
 
-![dominios](images/zonemaster04.png){.thumbnail}
+![Captura de pantalla de la página de resultados de Zonemaster para el dominio "mydomain.ovh". La sección "Address" se expande.](images/zonemaster04.png){.thumbnail}
 
 ### Información útil
 
-Si tiene más preguntas sobre Zonemaster, consulte [FAQ](https://zonemaster.net/es/faq) en <https://zonemaster.fr/>.
+Si tiene más preguntas sobre Zonemaster, consulte [FAQ](https://zonemaster.net/es/faq) en <https://zonemaster.net/>.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.es-es.md]:
```suggestion
Si tiene más preguntas sobre Zonemaster, consulte las [FAQ](https://zonemaster.net/es/faq) en <https://zonemaster.net/>.
```

File: pages/web/domains/dns_zonemaster/guide.es-us.md [hunk 3]
```diff
@@ -38,42 +36,45 @@ updated: 2022-09-12
...
+>
+> ![Captura de pantalla de la página de resultados de Zonemaster para el dominio "mydomain.ovh". Se desarrolla la sección "Direcciones".](images/zonemaster03.png){.thumbnail}
 
 ### Resultado
 
-Una vez validado el formulario, los resultados se clasifican por código de color:
+Una vez validado el formulario, los resultados se muestran por grupo de pruebas. Las pruebas se clasifican por su severidad.
+
+- **Error**: esta parte presenta errores o elementos faltantes que pueden causar un fallo de funcionamiento.
+- **Advertencia**: Esta parte es funcional, pero merece una atención especial. La herramienta ha detectado que este parámetro tiene características que no encajan en el estándar de su categoría, sin bloquear su funcionamiento.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.es-us.md]:
```suggestion
- **Advertencia**: esta parte es funcional, pero merece una atención especial. La herramienta ha detectado que este parámetro tiene características que no encajan en el estándar de su categoría, sin bloquear su funcionamiento.
```

File: pages/web/domains/dns_zonemaster/guide.es-us.md [hunk 4]
```diff
@@ -38,42 +36,45 @@ updated: 2022-09-12
...
-- **Azul**: se trata de una simple información, sin consecuencias particulares sobre el funcionamiento del nombre de dominio.
+Para cada prueba, es posible obtener más detalles, por ejemplo, para entender el error en caso de fallo, o simplemente a título indicativo.
 
-![dominios](images/zonemaster04.png){.thumbnail}
+![Captura de pantalla de la página de resultados de Zonemaster para el dominio "mydomain.ovh". La sección "Address" se expande.](images/zonemaster04.png){.thumbnail}
 
 ### Información útil
 
-Si tiene más preguntas sobre Zonemaster, consulte [FAQ](https://zonemaster.net/es/faq) en <https://zonemaster.fr/>.
+Si tiene más preguntas sobre Zonemaster, consulte [FAQ](https://zonemaster.net/es/faq) en <https://zonemaster.net/>.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.es-us.md]:
```suggestion
Si tiene más preguntas sobre Zonemaster, consulte las [FAQ](https://zonemaster.net/es/faq) en <https://zonemaster.net/>.
```

File: pages/web/domains/dns_zonemaster/guide.it-it.md [hunk 5]
```diff
@@ -38,39 +36,42 @@ updated: 2022-09-12
...
 ### I Risultati
 
 Una volta convalidato il modulo, i risultati sono classificati per codice colore:
 
-- **Verde**: Questa parte è funzionale e soddisfa i criteri standard nella sua categoria.
-- **Orange**: Questa sezione funziona, ma merita un'attenzione particolare. Il tool ha rilevato che questo parametro presenta caratteristiche che non rientrano nello standard della sua categoria, senza bloccarne il funzionamento.
-- **Rosso**: Questa parte presenta errori o elementi mancanti che possono causare un malfunzionamento. 
-- **Blu**: si tratta di una semplice informazione, senza conseguenze particolari sul funzionamento del dominio.
+- **Error**: questa parte presenta errori o elementi mancanti che possono causare malfunzionamenti.
+- **Warning**: Questa sezione funziona, ma merita un'attenzione particolare. Il tool ha rilevato che questo parametro presenta caratteristiche che non rientrano nello standard della sua categoria, senza bloccarne il funzionamento.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.it-it.md]:
```suggestion
- **Warning**: questa sezione funziona, ma merita un'attenzione particolare. Il tool ha rilevato che questo parametro presenta caratteristiche che non rientrano nello standard della sua categoria, senza bloccarne il funzionamento.
```

**Review by OVH-Team-Guides — COMMENTED** [review]
@tcpdumpfbacke , please review the DE version

File: pages/web/domains/dns_zonemaster/guide.es-es.md [hunk 6]
```diff
@@ -38,42 +36,46 @@ updated: 2022-09-12
...
+> ![Captura de pantalla de la página de resultados de Zonemaster para el dominio "mydomain.ovh". Se desarrolla la sección "Direcciones".](images/zonemaster03.png){.thumbnail}
 
 ### Resultado
 
-Una vez validado el formulario, los resultados se clasifican por código de color:
+Una vez validado el formulario, los resultados se muestran por grupo de pruebas. Las pruebas se clasifican por su severidad.
+
+- **Error**: esta parte presenta errores o elementos faltantes que pueden causar un fallo de funcionamiento.
+- **Advertencia**: Esta parte es funcional, pero merece una atención especial. La herramienta ha detectado que este parámetro tiene características que no encajan en el estándar de su categoría, sin bloquear su funcionamiento.
+- **Aviso**:  se trata de una simple información, sin consecuencias particulares sobre el funcionamiento del nombre de dominio.
```
Comment (bs-ovh) [inline pages/web/domains/dns_zonemaster/guide.es-es.md:68]: This is wrong, cf. EN/FR

File: pages/web/domains/dns_zonemaster/guide.it-it.md [hunk 7]
```diff
@@ -38,39 +36,42 @@ updated: 2022-09-12
...
+- **Info**: si tratta di una semplice informazione, senza conseguenze particolari sul funzionamento del dominio.
```
Comment (bs-ovh) [inline pages/web/domains/dns_zonemaster/guide.it-it.md]: This is wrong, cf. EN/FR

File: pages/web/domains/dns_zonemaster/guide.pt-pt.md [hunk 8]
```diff
@@ -38,43 +36,45 @@ updated: 2022-09-12
...
 
 Depois de validado o formulário, os resultados são classificados por código de cor:
 
-- **Verde**: Esta parte é funcional e cumpre os critérios standard na sua categoria.
-- **Laranja**: Esta parte é funcional, mas merece uma atenção especial. A ferramenta detetou que este parâmetro apresenta características que não correspondem ao padrão da sua categoria, sem bloquear o seu funcionamento.
-- **Vermelho**: Esta parte apresenta erros ou elementos em falta que podem causar uma anomalia. 
-- **Azul**: trata-se de uma simples informação, sem consequências particulares sobre o funcionamento do nome de domínio.
+- **Error**: esta parte apresenta erros ou elementos em falta que podem causar uma anomalia.
+- **Warning**: esta parte é funcional, mas merece uma atenção especial. A ferramenta detetou que este parâmetro apresenta características que não correspondem ao padrão da sua categoria, sem bloquear o seu funcionamento.
+- **Info**: trata-se de uma simples informação, sem consequências particulares sobre o funcionamento do nome de domínio.
```
Comment (bs-ovh) [inline pages/web/domains/dns_zonemaster/guide.pt-pt.md]: This is wrong, cf. EN/FR

File: pages/web/domains/dns_zonemaster/guide.pl-pl.md [hunk 9]
```diff
@@ -38,42 +36,44 @@ updated: 2022-09-12
...
 
 Po zatwierdzeniu formularza wyniki są klasyfikowane według kodu koloru:
 
-- **Zielony**: Część ta jest funkcjonalna i spełnia standardowe kryteria w swojej klasie.
-- **Pomarańczowy**: Ta część jest funkcjonalna, ale zasługuje na szczególną uwagę. Narzędzie wykryło, że ten parametr ma cechy, które nie wchodzą w zakres standardu jego kategorii bez zablokowania jego działania.
-- **Czerwony**: W tej części znajdują się błędy lub brakujące elementy, które mogą powodować usterkę. 
-- **Niebieski**: jest to zwykła informacja, bez wpływu na działanie domeny.
+- **Error**: ta część zawiera błędy lub brakujące elementy, które mogą powodować nieprawidłowe działanie.
+- **Warning**: ta część jest funkcjonalna, ale zasługuje na szczególną uwagę. Narzędzie wykryło, że parametr nie spełnia normy dla swojej kategorii, nie blokując przy tym jego działania.
+- **Info**: jest to prosta informacja, nie mająca szczególnego wpływu na funkcjonowanie nazwy domeny.
```
Comment (bs-ovh) [inline pages/web/domains/dns_zonemaster/guide.pl-pl.md]: This is wrong, cf. EN/FR

**Pull request merged**